Packages/Account/Sources/Account/AccountDetailViewModel.swift

```diff
@@ -129,7 +129,12 @@ class AccountDetailViewModel: ObservableObject, StatusesFetcher {
 
   private func fetchAccountData(accountId: String, client: Client) async throws -> AccountData {
     async let account: Account = client.get(endpoint: Accounts.accounts(id: accountId))
-    async let featuredTags: [FeaturedTag] = client.get(endpoint: Accounts.featuredTags(id: accountId))
+    let featuredTags: [FeaturedTag]
+    do {
```

Comment:
You can't fix it like that, it'll break the concurrency, well basically it'll first await this call and then do the second one. I'll look to do it another way.

Reply:
Thank you for your code review.
Fixed to a different approach that does not cause errors in processing parallelism.
how about this?

Reply:
Great thanks!